src/System.Net.Requests/tests/project.json

```diff
@@ -22,13 +22,32 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00807-03"
   },
   "frameworks": {
-    "netstandard1.3": {}
+    "netstandard1.3": { },
+    "netstandard1.7": { }
   },
   "supports": {
-    "coreFx.Test.netcoreapp1.0": {},
-    "coreFx.Test.net46": {},
-    "coreFx.Test.net461": {},
-    "coreFx.Test.net462": {},
-    "coreFx.Test.net463": {}
+    "coreFx.Test.netcoreapp1.0": { },
+    "coreFx.Test.net46": { },
+    "coreFx.Test.net461": { },
+    "coreFx.Test.net462": { },
+    "coreFx.Test.net463": { },
+    "coreFx.Test.netcoreapp1.1-ns17": {
```

Comment:
This whole section should be able to be turned into "corefx.Test.netcoreapp1.1" now.